docs/docs-content/troubleshooting/cluster-deployment.md

```diff
@@ -94,6 +95,31 @@ Common reasons for why a service may fail are:
 
 6. Check stdout for errors. You can also open a support ticket. Visit our [support page](http://support.spectrocloud.io/).
 
+## Deployment Violates Pod Security
+In cluster profiles running Kubernetes 1.25 or later, some packs contain Pods that need elevated privileges during Pod creation. 
+When you try to deploy a cluster with such packs, you get an error message that looks like the following:
+
+```
+Error creating: pods <name of pod> is forbidden: violates PodSecurity "baseline:v<k8s version>": non-default capabilities …
+```
+
+### Debug Steps
+
+You can change the Pod Security Standards of the namepace where the Pod is being created to address this issue. 
+
+1. Log in to **Palette** and navigate to the **Profiles** tab. 
+2. Select the profile you are trying to deploy the cluster with and choose the layer that represents your pack. 
+3. In the YAML file for your pack, under the `pack` field, add a subfield `namespaceLabels`.
```

Comment:
```suggestion
3. In the `pack` section of the Kubernetes YAML file, add a subsection called `namespaceLabels`.
```

Reply:
Lenny, as you and I discussed, it might be _other_ packs in combination with Kubernetes packs (1.25 and higher).